CHANGELOG/CHANGELOG-1.10.md

```diff
@@ -0,0 +1,74 @@
+    * Refer to [this document](https://github.com/antrea-io/antrea/blob/release-1.10/docs/support-bundle-guide.md) for more information about this feature.
+- Add support for NetworkPolicy for Cross-cluster Traffic. ([#4432](https://github.com/antrea-io/antrea/pull/4432) [#3914](https://github.com/antrea-io/antrea/pull/3914), [@Dyanngg] [@GraysonWu])
+    * Setting `scope` of an ingress peer to `clusterSet` expands the scope of the `podSelector` or `namespaceSelector` to the entire ClusterSet.
+    * Setting `scope` of `toServices` to `clusterSet` selects a Multi-cluster Service. ([#4397](https://github.com/antrea-io/antrea/pull/4397), [@Dyanngg])
+    * Refer to [this document](https://github.com/antrea-io/antrea/blob/release-1.10/docs/multicluster/user-guide.md#networkpolicy-for-cross-cluster-traffic) for more information about this feature.
+- Add support for containerd runtime for antrea-agent on Windows. ([#4279](https://github.com/antrea-io/antrea/pull/4279), [@XinShuYang])
+- Add the following capabilities to the ExternalNode feature:
+    * Add the containerize option for antrea-agent installation on Linux VMs . ([#4413](https://github.com/antrea-io/antrea/pull/4413), [@Nithish555])
+    * Add support for RHEL OS. ([#4323](https://github.com/antrea-io/antrea/pull/4323), [@Nithish555])
+- Add [documentation](https://github.com/antrea-io/antrea/blob/release-1.10/docs/multicast-guide.md) for Antrea Multicast. ([#4339](https://github.com/antrea-io/antrea/pull/4339), [@ceclinux])
+
+### Changed
+
+- Extend `antctl mc get joinconfig` to print member token Secret. ([#4363](https://github.com/antrea-io/antrea/pull/4363), [@jianjuns])
+- Improve support for Egress in Traceflow. ([#3926](https://github.com/antrea-io/antrea/pull/3926), [@Atish-iaf])
+- Add Antrea agent configuration options to enable and configure secondary network specific OVS parameters. ([#4440](https://github.com/antrea-io/antrea/pull/4440), [@adwaitni])
+- Add NodePortLocalRange field for agentinfo. ([#4379](https://github.com/antrea-io/antrea/pull/4379), [@wenqiq])
+- Use format "namespace/name" as the key for ExternalNode span calculation. ([#4401](https://github.com/antrea-io/antrea/pull/4401), [@wenyingd])
+- Enclose Pod labels with single quote in s3 uploader for Snowflake use cases. ([#4334](https://github.com/antrea-io/antrea/pull/4334), [@dreamtalen])
+- Upgrade Antrea base image to ubuntu 22.04. ([#4459](https://github.com/antrea-io/antrea/pull/4459) [#4499](https://github.com/antrea-io/antrea/pull/4499), [@antoninbas])
+- Update OVS to 2.17.3. ([#4402](https://github.com/antrea-io/antrea/pull/4402), [@mnaser])
+- Reduce confusion caused by transient error encountered creating tiers. ([#4414](https://github.com/antrea-io/antrea/pull/4414), [@tnqn])
+
+### Fixed
+
+- Add a periodic job to rejoin dead Nodes to fix Egress not working properly after long network downtime. ([#4491](https://github.com/antrea-io/antrea/pull/4491), [@tnqn])
```

Comment:
s/to rejoin dead Nodes to fix Egress/to rejoin dead Nodes, to fix Egress

Reply:
done